### PR TITLE
Added Magical Gifts

### PR DESCRIPTION
**Pull request by Batziy**
Added Magical Gifts from Tome of Chaos.

### CODE REVIEW

**Comment by Batziy** [comment]
#resolves #251

File: scripts/TheWitcherTRPG.js [hunk 1]
```diff
@@ -27,7 +27,9 @@ async function preloadHandlebarsTemplates(){
         "systems/TheWitcherTRPG/templates/partials/skill-display.html",
         "systems/TheWitcherTRPG/templates/partials/monster-skill-display.html",
         "systems/TheWitcherTRPG/templates/partials/loot-item-display.html",
-        "systems/TheWitcherTRPG/templates/partials/item-header.html"
+        "systems/TheWitcherTRPG/templates/partials/item-header.html",
+        "systems/TheWitcherTRPG/templates/sheets/verbal-combat.html",
+        "systems/TheWitcherTRPG/templates/sheets/weapon-attack.html"
```
[the remark this answers is not in the document]
Comment (Batziy) [reply scripts/TheWitcherTRPG.js:32]: Yes im currently working on the weapon-attack one :)

[the remark this answers is not in the document]
Comment (AnthonyMonette) [reply scripts/TheWitcherTRPG.js:32]: Okay should I wait for those changes before merging?

[the remark this answers is not in the document]
Comment (AnthonyMonette) [reply scripts/TheWitcherTRPG.js:32]: And thank you so much, I'm waiting to receive my copy of Thome Of Cahos,  that's why I didn't addressed this issue yet

[the remark this answers is not in the document]
Comment (Batziy) [reply scripts/TheWitcherTRPG.js:32]: Hehe okey :P Might take some time. I have some problem with the weapon attack sheet. You can either omit it or include it for now  :P Shouldn't break something anyways.

File: module/sheets/WitcherActorSheet.js [hunk 2]
```diff
@@ -1605,34 +1609,9 @@ export default class WitcherActorSheet extends ActorSheet {
...
-          <div><input name="verbalCombat" type="radio" id="Study" value="Study"/><label for="Study">${game.i18n.localize("WITCHER.verbalCombat.Study")}</label></div>
-          <h2>${game.i18n.localize("WITCHER.verbalCombat.AntagonisticTools")}</h2>
-          <div><input name="verbalCombat" type="radio" id="ImplyPersuade" value="ImplyPersuade"/><label for="ImplyPersuade">${game.i18n.localize("WITCHER.verbalCombat.ImplyPersuade")}</label></div>
-          <div><input name="verbalCombat" type="radio" id="ImplyDeceit" value="ImplyDeceit"/><label for="ImplyDeceit">${game.i18n.localize("WITCHER.verbalCombat.ImplyDeceit")}</label></div>
-          <div><input name="verbalCombat" type="radio" id="Bribe" value="Bribe"/><label for="Bribe">${game.i18n.localize("WITCHER.verbalCombat.Bribe")}</label></div>
-          <label>${game.i18n.localize("WITCHER.Dialog.attackCustom")}: <input name="customModifiers" value=0></label>
-        </div>`;
+      const dialogTemplate = await renderTemplate("systems/TheWitcherTRPG/templates/sheets/verbal-combat.html");
```
Comment (AnthonyMonette) [inline module/sheets/WitcherActorSheet.js:1614]: That's nice I didn't know we could do that! Good thing to know

**Pull request merged**